Harvest.Jobs.Invoice/Program.cs

```diff
@@ -49,7 +80,12 @@ private static ServiceProvider ConfigureServices()
 #endif
             });
 
+            services.Configure<SlothSettings>(Configuration.GetSection("Sloth"));
+            services.Configure<FinancialLookupSettings>(Configuration.GetSection("FinancialLookup"));
+
             services.AddTransient<IInvoiceService, InvoiceService>();
+            services.AddTransient<ISlothService, SlothService>();
+            services.AddTransient<IFinancialService, FinancialService>();

```

Comment:
This is kinda a fun point, and yes you are correct that multiple instances will be created.  the service doesn't keep it's own state though and so it's not actually a problem, save possible a few extra allocations.  BUT that said, we could probably improve it in 2 ways -- first and biggest, we could take HttpClient as a parameter and use the durable httpClient factory to make sure that our client didn't need to be created/released every time.  second, we could make it a singleton instead of transient and that would save possible a few bytes here or there.